xml/System.Reflection.PortableExecutable/PEHeader.xml

```diff
@@ -150,8 +162,8 @@ For program images, this is the starting address. For device drivers, this is th
         <ReturnType>System.Reflection.PortableExecutable.DirectoryEntry</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the Base relocations table entry.</summary>
```

Comment:
I'm unsure about the capitalization here and throughout this file. I see that the field is named "Base Relocations Table" at https://learn.microsoft.com/en-us/windows/win32/debug/pe-format#optional-header-data-directories-image-only. So either all three words should be capitalized, or none of them.

Reply:
Changed to capitalize all words in entry names, as preexisting Certificates Table used this style.